tsqa/environment.py

```diff
@@ -435,19 +455,28 @@ def destroy(self):
         installed files.
         """
         self.stop()
-        shutil.rmtree(self.layout.prefix, ignore_errors=True)
+        if self.keep_tmp is False:
```

Comment:
This keep_tmp option shouldn't stop destroy() from deleting, we should change the base EnvironmentCase from calling destroy-- since destroy is... well destructive by design :)